Documentation: Add information about database adapter `dbt-cratedb2`

amotl opened the pull request:
## About
This is the documentation page to be submitted to dbt's documentation space, like the [Postgres setup] page. It has been made similar like the documentation pages [AlloyDB setup], [Greenplum setup], and [Yellowbrick setup].

## Preview
[Configuration for dbt CrateDB adapter](https://github.com/crate-workbench/docs.getdbt.com/blob/cratedb/website/docs/docs/core/connect-data-platform/cratedb-setup.md) (GitHub-rendered)

## Call for Support
_This is the right spot to share your voice on a remote documentation page._

The page needs a bit of your support to finalize, in terms of user testing and polishing the docs. There is not much inside, so it will be nice to get the ingredients right from the very beginning.

/cc @karynzv, @hlcianfagna, @hammerhead, @surister, @kneth, @simonprickett, @proddata, @geragray 

[AlloyDB setup]: https://docs.getdbt.com/docs/core/connect-data-platform/alloydb-setup
[Greenplum setup]: https://docs.getdbt.com/docs/core/connect-data-platform/greenplum-setup
[Postgres setup]: https://docs.getdbt.com/docs/core/connect-data-platform/postgres-setup
[Yellowbrick setup]: https://docs.getdbt.com/docs/core/connect-data-platform/yellowbrick-setup

amotl committed:
Resource configurations: Improve `no-configs` page

- Fix a typo s/specifc/specific/
- Add third variant why no documentation about platform-specific
  configuration settings is provided, by referring to vendor
  documentation pages

amotl commented on website/docs/docs/core/connect-data-platform/cratedb-setup.md:
````diff
@@ -0,0 +1,75 @@
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+company-name:
+  target: dev
+  outputs:
+    dev:
+      type: cratedb
+      host: [hostname]
+      user: [username]
+      password: [password]
+      port: [port]
+      dbname: [database name] # or database instead of dbname
+      schema: [dbt schema]
+      threads: [optional, 1 or more]
+      [keepalives_idle](#keepalives_idle): 0 # default 0, indicating the system default. See below
+      connect_timeout: 10 # default 10 seconds
+      [retries](#retries): 1  # default 1 retry on error/timeout when opening connections
+      [search_path](#search_path): [optional, override the default postgres search_path]
+      [role](#role): [optional, set the role dbt assumes when executing queries]
+      [sslmode](#sslmode): [optional, set the sslmode used to connect to the database]
+      [sslcert](#sslcert): [optional, set the sslcert to control the certifcate file location]
+      [sslkey](#sslkey): [optional, set the sslkey to control the location of the private key]
+      [sslrootcert](#sslrootcert): [optional, set the sslrootcert config value to a new file path in order to customize the file location that contain root certificates]
+  
+```
````
That's the vanilla section of connection options copied from `postgres-setup.md`, roughly like AlloyDB, Greenplum and Yellowbrick are also doing it, because they also build upon dbt-postgres. However, a few of them omit some options they do not support. We should probably do the same.

Because some of you may have used psycopg2 together with CrateDB in more environments than me already, it might be easy for you to quickly identify the options we should remove right away. Thanks!

amotl replied:
> Let's not duplicate too much information here, it is easier to update documentation on our premises if we decide to change something.

Right. I've trimmed this section to include only a minimal set of parameters.

amotl commented on website/docs/docs/core/connect-data-platform/cratedb-setup.md:
```diff
@@ -0,0 +1,75 @@
+
+</File>
+
+### Configuration notes
+
+This adapter strongly depends on dbt-postgres documented at [Postgres profile setup](/docs/core/connect-data-platform/postgres-setup). CrateDB targets are configured exactly the same way.
+
+
+### Configuration differences
+
+#### search_path
+
+The `search_path` config controls the CrateDB "search path" that dbt configures
+when opening new connections to the database. By default, the CrateDB search
+path is `"doc"`, meaning that unqualified <Term id="table" /> names will be
+searched for in the `doc` schema.
```
That little section at the bottom outlines the **typical differences** of CrateDB. Please also share your opinion about what you would put in here.

amotl replied:
Maybe any of those would also be relevant to replicate here, in order to educate users approaching CrateDB the very first time from the dbt perspective?

- CrateDB’s fixed catalog name is `crate`, the default schema name is `doc`.
- CrateDB does not implement the notion of a database, however tables can be created in different [schemas](https://cratedb.com/docs/crate/reference/en/latest/general/ddl/create-table.html#ddl-create-table-schemas).
- When asked for a database name, specifying a schema name (any), or the fixed catalog name `crate` may be applicable.
- If a database-/schema-name is omitted while connecting, the PostgreSQL drivers may default to the “username”.
- The predefined [superuser](https://cratedb.com/docs/crate/reference/en/latest/admin/user-management.html#administration-user-management) on an unconfigured CrateDB cluster is called `crate`, defined without a password.
- For authenticating properly, please learn about the available [authentication](https://cratedb.com/docs/crate/reference/en/latest/admin/auth/index.html#admin-auth) options.

-- https://cratedb.com/docs/crate/clients-tools/en/latest/connect/#configure

kneth replied:
Let's not duplicate the information and simply have the link to our documentation (easier to update if we decide to change something)

amotl replied:
I absolutely agree. Let's do it like [Greenplum setup](https://docs.getdbt.com/docs/core/connect-data-platform/greenplum-setup) is doing it, by adding such a paragraph:

> For further and more likely up-to-date information, see the documentation about [using dbt with CrateDB](https://cratedb.com/docs/guide/integrate/dbt/).

amotl replied:
I've removed all too specific information, and will be adding it to [using dbt with CrateDB](https://cratedb.com/docs/guide/integrate/dbt/) instead. I've also updated the document body to include a relevant link to that page.

amotl reviewed (commented):
I've added a few questions per inline comments. Thanks!

amotl commented on website/docs/docs/core/connect-data-platform/cratedb-setup.md:
```diff
@@ -0,0 +1,75 @@
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Supported
```
Whether `cloud_support: Supported` is indeed true, we need to validate. I would like to defer or delegate relevant investigations, please pick up the torch.

- https://github.com/crate/devrel/issues/58

amotl replied:
We learned from @hlcianfagna that this flag may only be toggled by people of dbt, because only popular and requested database plugins will gain "dbt Cloud support"? I am not sure about the semantics of this particular attribute yet.

amotl replied:
I've not changed it. Let's see what the outcome from the official review will be.

amotl commented on website/docs/docs/core/connect-data-platform/cratedb-setup.md:
```diff
@@ -0,0 +1,75 @@
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Community
```
@geragray said:

> Going through criteria for trusted adapters, it seems like it is fairly easy to [get in](https://github.com/crate/devrel/issues/57#user-content-fn-1-af26b626dd624c30511ff8438c652515). It just needs to work plus maintained actively (by a company, not a community).

amotl replied:
I've used `Crate.io, Inc.` here, now.

faithebear committed:
Update 2024-11-27-test-smarter-part-2--add diagram

mirnawong1 committed:
remove beta

faithebear committed:
first pass at formatting edits to test smarter part 2

faithebear committed:
Update 2024-11-27-test-smarter-part-2 info box

matthewshaver committed:
Merge branch 'current' into mckenna-kenney-test-smarter-part-2

matthewshaver committed:
Update website/blog/2024-11-27-test-smarter-part-2

matthewshaver committed:
Update website/blog/2024-11-27-test-smarter-part-2

matthewshaver committed:
Adding file extension

faithebear committed:
Update 2024-11-27-test-smarter-part-2.md

formatting updates

faithebear committed:
Update 2024-11-27-test-smarter-part-2.md img path

faithebear committed:
Update 2024-11-27-test-smarter-part-2.md even more formatting

amotl commented on website/docs/docs/core/connect-data-platform/cratedb-setup.md:
```diff
@@ -0,0 +1,75 @@
+
+### Configuration notes
+
+This adapter strongly depends on dbt-postgres documented at [Postgres profile setup](/docs/core/connect-data-platform/postgres-setup). CrateDB targets are configured exactly the same way.
```
Another proposal where/how to fluently refer to the docs on our premises.

```suggestion
This adapter strongly depends on dbt-postgres documented at [Postgres profile setup](/docs/core/connect-data-platform/postgres-setup). CrateDB targets are configured exactly the same way with just a few differences outlined on [using dbt with CrateDB](https://cratedb.com/docs/guide/integrate/dbt/).
```

amotl replied:
Done.

amotl commented:
The patch has been submitted to upstream.

- https://github.com/dbt-labs/docs.getdbt.com/pull/6564

_Please don't close or merge this PR, otherwise the underlying branch would be deleted, which has detrimental effects on the upstream PR._

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

mirnawong1 committed:
add integartions page

mirnawong1 committed:
add rn

mirnawong1 committed:
add beta flag

mirnawong1 committed:
add steps

mirnawong1 committed:
add links

mirnawong1 committed:
add enterprise pill

mirnawong1 committed:
rephrase

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

faithebear committed:
Delete website/static/img/blog/2024-11-27-test-smarter-part-2/testing_pipeline.png

delete old testing image

faithebear committed:
Create beep.md

faithebear committed:
add improved testing_pipeline diagram

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into add-ai-integration-page

runleonarun committed:
Merge branch 'current' into mckenna-kenney-test-smarter-part-2

faithebear committed:
Delete website/static/img/blog/2024-11-27-test-smarter-part-2/beep.md

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

faithebear committed:
adding truncate

faithebear committed:
fixing advanced CI link

faithebear committed:
changing up examples of business-focused anomalies

faithebear committed:
updating source freshness guidance to be less cloud-y

faithebear committed:
used code tags wrong

faithebear committed:
fixing even more formatting

faithebear committed:
fixing up staging examples

runleonarun committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Merge branch 'current' into add-ai-integration-page

mirnawong1 committed:
add callout for collections synching

this pr turns the existing content about the once a day sync into a callout and adds info on how to trigger a manual sync per [slack thread](https://dbt-labs.slack.com/archives/C06GJRK27PF/p1733359577763749) and user confusion.

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
Update configure-auto-exposures.md

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

faithebear committed:
a few formatting and light clarity changes

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

matthewshaver committed:
Fixing broken links

matthewshaver committed:
Enhancing BigQuery language

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-26

matthewshaver committed:
Adding folders to dbtignore

joellabes committed:
Merge branch 'current' into mckenna-kenney-test-smarter-part-2

faithebear committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

dbeatty10 committed:
Link to `store_failures_as` config instead of `--store-failures` flag

dbeatty10 committed:
Add caveat about including all necessary given input fixtures for unit tests

dbeatty10 committed:
Link to docs for `model-paths` configuration

dbeatty10 committed:
dbt looks for unit tests in the directories in the `model-paths` config

dbeatty10 committed:
Separate adapter-specific caveats for unit tests

mirnawong1 committed:
Fixing broken links (#6598)

mirnawong1 committed:
Merge branch 'current' into bigquery-language

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-1

mirnawong1 committed:
Enhancing BigQuery language (#6599)

mirnawong1 committed:
Update website/docs/docs/build/unit-tests.md

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-1

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-2

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-3

mirnawong1 committed:
Update website/docs/reference/project-configs/model-paths.md

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-5

mirnawong1 committed:
Update website/docs/docs/build/unit-tests.md

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-6

mirnawong1 committed:
Link to `store_failures_as` config instead of `--store-failures` flag (#6602)

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

Co-authored-by: Joel Labes <joel.labes@dbtlabs.com>

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-2

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-3

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-5

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-6

mirnawong1 committed:
Add caveat about including all necessary given input fixtures for unit tests (#6603)

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-5

mirnawong1 committed:
Merge branch 'current' into dbtignore

mirnawong1 committed:
dbt looks for unit tests in the directories in the `model-paths` config (#6605)

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-6

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-6

mirnawong1 committed:
Merge branch 'current' into dbtignore

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-3

mirnawong1 committed:
Separate adapter-specific caveats for unit tests (#6606)

mirnawong1 committed:
Merge branch 'current' into dbtignore

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-3

mirnawong1 committed:
Adding folders to dbtignore (#6600)

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-3

mirnawong1 committed:
Link to docs for `model-paths` configuration (#6604)

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Merge branch 'current' into add-ai-integration-page

mirnawong1 committed:
multiple emails not supported

mirnawong1 committed:
small tweaks

mirnawong1 committed:
add link

mirnawong1 committed:
format

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
update

mirnawong1 committed:
Merge branch 'add-ai-integration-page' of github.com:dbt-labs/docs.getdbt.com into add-ai-integration-page

mirnawong1 committed:
adjust rn

mirnawong1 committed:
add managed

mirnawong1 committed:
Merge branch 'current' into mckenna-kenney-test-smarter-part-2

mirnawong1 committed:
Update website/blog/2024-11-27-test-smarter-part-2.md

mirnawong1 committed:
small tweak

mirnawong1 committed:
small tweak

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
Update website/docs/docs/cloud/account-integrations.md

mirnawong1 committed:
add new integration page (#6579)

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

QMalcolm committed:
Add resource-config pages for microbatch configs (#6575)

## What are you changing in this pull request and why?
We had resource config page for `event_time` but not, `begin`,
`lookback`, or `batch_size` (which are all configs for microbatch
incremental models). This seemed like a gap. As such, I've added pages
for `begin`, `lookback`, and `batch_size` and also linked them in the
incremental-microbatch page.

## Checklist
- [ ] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I
have versioned it according to the [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and/or [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)
guidelines.
- [ ] I have added checklist item(s) to this list for anything anything
that needs to happen before this PR is merged, such as "needs technical
review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one
to the [release notes
page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages
-->

---------

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>
Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Merge branch 'current' into mckenna-kenney-test-smarter-part-2

mirnawong1 committed:
Create 2024-11-27-test-smarter-part-2--initial commit (#6559)

dataders committed:
Update website/docs/docs/core/connect-data-platform/mssql-setup.md

dataders committed:
Merge branch 'current' into patch-2

mirnawong1 committed:
Update mssql-setup.md

rendering incorrectly

mirnawong1 committed:
Update website/docs/docs/core/connect-data-platform/mssql-setup.md

mirnawong1 committed:
Update mssql-setup.md (#6304)

mirnawong1 committed:
Update model-notifications.md

rephrase and clarify per Rakesh's feedback

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-26

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

mirnawong1 committed:
add fields

dbeatty10 committed:
Remove link to customizing the name of a model's database identifier

mirnawong1 committed:
Update website/docs/docs/build/simple.md

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Update model-notifications.md

mirnawong1 committed:
Update simple.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update simple.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

joellabes committed:
Swap latest callouts for fka versionless inline (#6607)

## What are you changing in this pull request and why?
[As discussed in
Slack](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1733449411561619?thread_ts=1733149843.050859&cid=C02NCQ9483C)
- swap callouts for a more inline explanation of the changes, for blog
posts where Versionless isn't the main focus so the rename doesn't
matter as much.

## Checklist
- [x] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.

dbeatty10 committed:
Explain behavior of `config.get`

nataliefiann committed:
Created new section called Parallel batch execution (#6589)

## What are you changing in this pull request and why?

I have created this PR following this Git issue:
https://github.com/dbt-labs/docs.getdbt.com/issues/6550 for Parallel
batch execution

## Checklist
- [ ] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I
have versioned it according to the [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and/or [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)
guidelines.
- [ ] I have added checklist item(s) to this list for anything anything
that needs to happen before this PR is merged, such as "needs technical
review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one
to the [release notes
page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages
-->

<!-- vercel-deployment-preview -->
---
🚀 Deployment available! Here are the direct links to the updated files:


-
https://docs-getdbt-com-git-nfiann-rbip-dbt-labs.vercel.app/docs/build/incremental-microbatch
-
https://docs-getdbt-com-git-nfiann-rbip-dbt-labs.vercel.app/docs/dbt-versions/core-upgrade/06-upgrading-to-v1.9
-
https://docs-getdbt-com-git-nfiann-rbip-dbt-labs.vercel.app/reference/resource-properties/concurrent_batches

<!-- end-vercel-deployment-preview -->

---------

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>
Co-authored-by: Quigley Malcolm <QMalcolm@users.noreply.github.com>
Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-2

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Update website/docs/reference/dbt-jinja-functions/config.md

mirnawong1 committed:
Explain behavior of `config.get` (#6611)

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-1

mirnawong1 committed:
Update website/docs/docs/build/simple.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/simple.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into update-simple-metrics

mirnawong1 committed:
Update website/docs/docs/cloud-integrations/configure-auto-exposures.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-26

mirnawong1 committed:
Remove link to customizing the name of a model's database identifier using the `alias` model config (#6610)

mirnawong1 committed:
Merge branch 'current' into update-simple-metrics

mirnawong1 committed:
Update configure-auto-exposures.md

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-26

nataliefiann committed:
Updated prod environment image

mirnawong1 committed:
add links to configs

mirnawong1 committed:
update availability

mirnawong1 committed:
Update incremental-strategy.md

fabric doesn't support the `merge` strategy per here: https://learn.microsoft.com/en-us/fabric/data-warehouse/tsql-surface-area#limitations

refer to [internal slack](https://dbt-labs.slack.com/archives/C067EUS2D7S/p1733718174151849)

nataliefiann committed:
Updated prod environment image (#6615)

mirnawong1 committed:
Merge branch 'current' into add-config-links

mirnawong1 committed:
add links to microbatch configs (#6616)

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-27

mirnawong1 committed:
Update incremental-strategy.md (#6618)

nataliefiann committed:
Updated CLI doc to provide info on stuck session error

nataliefiann committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

nataliefiann committed:
Updated CLI doc to provide info on stuck session error (#6620)

ramiz-bozai committed:
Update databricks-privatelink.md

Removed typo

mirnawong1 committed:
Update databricks-privatelink.md (#6623)

dbeatty10 committed:
Add unit tests limitations section for BigQuery (for `STRUCT`)

john-rock committed:
add strict csp policy

john-rock committed:
update more resources

john-rock committed:
update wistia

matthewshaver committed:
GA v1.9 in docs (#6626)

## What are you changing in this pull request and why?

Updates docs for v1.9 GA


## Checklist
- [ ] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I
have versioned it according to the [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and/or [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)
guidelines.
- [ ] I have added checklist item(s) to this list for anything anything
that needs to happen before this PR is merged, such as "needs technical
review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one
to the [release notes
page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages
-->

john-rock committed:
url updates

john-rock committed:
revert all for testing

dbeatty10 committed:
Merge branch 'current' into dbeatty10-patch-1

matthewshaver committed:
Fixing syntax (#6627)

## What are you changing in this pull request and why?

Closes https://github.com/dbt-labs/docs.getdbt.com/issues/5394

## Checklist
- [ ] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I
have versioned it according to the [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and/or [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)
guidelines.
- [ ] I have added checklist item(s) to this list for anything anything
that needs to happen before this PR is merged, such as "needs technical
review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one
to the [release notes
page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages
-->

<!-- vercel-deployment-preview -->
---
🚀 Deployment available! Here are the direct links to the updated files:


-
https://docs-getdbt-com-git-indirect-dbt-labs.vercel.app/reference/global-configs/indirect-selection

<!-- end-vercel-deployment-preview -->

matthewshaver committed:
Altering syntax

mirnawong1 committed:
Merge branch 'current' into mwong-model-level-ga

mirnawong1 committed:
Update model-notifications.md

mirnawong1 committed:
update model level to ga (#6581)

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-26

mirnawong1 committed:
add callout for collections synching (#6594)

mirnawong1 committed:
Merge branch 'current' into update-simple-metrics

mirnawong1 committed:
add alias/filter fields to simple and converesion metrics (#6609)

mirnawong1 committed:
Merge branch 'current' into subdirectory

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-1

mirnawong1 committed:
Altering syntax (#6630)

mirnawong1 committed:
add rn entry

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

mirnawong1 committed:
Merge branch 'current' into dbeatty10-patch-1

mirnawong1 committed:
Add unit tests limitations section for BigQuery (for `STRUCT`) (#6625)

mirnawong1 committed:
Merge branch 'current' into sl-rn-execution-project

nataliefiann committed:
Updated images on environmental variable page

mirnawong1 committed:
add two new faqs

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

mirnawong1 committed:
change header

mirnawong1 committed:
Update website/docs/faqs/Troubleshooting/error-importing-repo.md

Co-authored-by: nataliefiann <120089939+nataliefiann@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/faqs/Troubleshooting/gitlab-webhook.md

Co-authored-by: nataliefiann <120089939+nataliefiann@users.noreply.github.com>

mirnawong1 committed:
add gitlab webhook faq (#6635)

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

mirnawong1 committed:
update unique_key

mirnawong1 committed:
Merge branch 'current' into sl-rn-execution-project

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

mirnawong1 committed:
add rn entry for execution_project (#6631)

nataliefiann committed:
Updated mesh doc to add new prod environment img

mirnawong1 committed:
Update website/docs/reference/resource-configs/unique_key.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/reference/resource-configs/unique_key.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/reference/resource-configs/unique_key.md

Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into update-unique-key

mirnawong1 committed:
move unique_key page under general configs (#6636)

john-rock committed:
add frame ancestors

john-rock committed:
add frame ancestors

john-rock committed:
update mutiny domain

john-rock committed:
Merge branch 'current' into update-strict-csp

amotl committed:
Platform connections: Add new data platform adapter for CrateDB

The Python package is called `dbt-cratedb2`.

Package: https://pypi.org/project/dbt-cratedb2/
Repository: https://github.com/crate/dbt-cratedb2
Documentation: https://cratedb.com/docs/guide/integrate/dbt/

mirnawong1 committed:
Merge branch 'current' into nfiann-mesh-img

mirnawong1 committed:
Updated mesh doc to add new prod environment img (#6637)

mirnawong1 committed:
Update get-started-dbt.md

visualize core quickstarts into cards per feedback: https://github.com/dbt-labs/docs.getdbt.com/issues/6638

mirnawong1 committed:
Update website/docs/docs/get-started-dbt.md

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

mirnawong1 committed:
Update get-started-dbt.md (#6639)

mirnawong1 committed:
Update get-started-dbt.md

remove backticks and committing to current directly

nataliefiann committed:
Merge branch 'current' into nfiann-img-sweep

nataliefiann committed:
Updated images on environmental variable page (#6634)

mirnawong1 committed:
add new content

mirnawong1 committed:
Update website/docs/docs/cloud/about-cloud/about-dbt-cloud.md

mirnawong1 committed:
udpates

mirnawong1 committed:
Merge branch 'dbt-copilot-sql' of github.com:dbt-labs/docs.getdbt.com into dbt-copilot-sql

mirnawong1 committed:
Update website/docs/docs/cloud/use-dbt-copilot.md

mirnawong1 committed:
Update website/docs/docs/cloud/use-dbt-copilot.md

Co-authored-by: dave-connors-3 <73915542+dave-connors-3@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/cloud/use-dbt-copilot.md

mirnawong1 committed:
Update use-dbt-copilot.md

mirnawong1 committed:
add new inline sql generation section (#6642)

jtcohen6 committed:
Update "OSS Expectations" (#6551)

## What are you changing in this pull request and why?

Follow-up to:
- https://github.com/dbt-labs/docs.getdbt.com/pull/5584
- https://www.youtube.com/watch?v=DC9sbZBYzpI
- https://www.youtube.com/watch?v=I72yUtrmhbY

Let's make sure our public expectations are aligned with our teams'
commitments around responding to OSS discussions/issues and reviewing
externally contributed PRs

---------

Co-authored-by: graciegoheen <graciegoheen@gmail.com>
Co-authored-by: Doug Beatty <44704949+dbeatty10@users.noreply.github.com>
Co-authored-by: Matt Shaver <60105315+matthewshaver@users.noreply.github.com>

john-rock committed:
Merge branch 'current' into update-strict-csp

john-rock committed:
Update CSP policy (#6629)

runleonarun committed:
Update incremental-microbatch.md (#6644)

## What are you changing in this pull request and why?

Per this Slack discussion we want to add more context to the
concurrent_batches flag to make sure people understand it's an override
to the auto detect and not the detection itself.


## Checklist
- [ ] I have reviewed the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I
have versioned it according to the [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and/or [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)
guidelines.
- [ ] I have added checklist item(s) to this list for anything anything
that needs to happen before this PR is merged, such as "needs technical
review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one
to the [release notes
page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).
<!--
PRE-RELEASE VERSION OF dbt (if so, uncomment):
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
<!-- 
ADDING OR REMOVING PAGES (if so, uncomment):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages
-->

<!-- vercel-deployment-preview -->
---
🚀 Deployment available! Here are the direct links to the updated files:


-
https://docs-getdbt-com-git-runleonarun-patch-15-dbt-labs.vercel.app/docs/build/incremental-microbatch

<!-- end-vercel-deployment-preview -->

amotl commented:
> The patch has been submitted to upstream.
>
> - https://github.com/dbt-labs/docs.getdbt.com/pull/6564

The patch has been merged.

https://docs.getdbt.com/docs/core/connect-data-platform/cratedb-setup
